src/gui/QvisPlotManagerWidget.C

```diff
@@ -1085,6 +1098,7 @@ QvisPlotManagerWidget::UpdatePlotList()
 
         prefixes.push_back(prefix);
     }
+    
```

Comment:
Not sure what an additional blank line here improves. seems like 1 blank line here would be sufficient.

Reply:
Doesn't improve anything @brugger1, just a typo. I removed the extra new line.